app/process-image.js

```diff
@@ -793,9 +812,9 @@ class ImageProcessing {
 
 			// basic level 0 processing by default
 			if (level === 0) {
-				new_image = new_image.scan(0, 0, region.width, region.height, this.filterBodyContent).blur(1);
```

Comment:
This change is debatable, I found the blurring in this manor often improved results, did you see any scenarios where it was improved by NOT doing this?

Reply:
Yes, results are absolutely more reliable without blurring; quite a few characters get blurred into enough of oblivion that tesseract can't reliably pick them up even with a blur pixel size of 1, and taking the blurring out doesn't cause it to pick up non-characters as one any more frequently.